src/muiThemeable.js

```diff
@@ -1,23 +1,39 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
+import merge from 'lodash.merge';
+
+const defaultTheme = getMuiTheme();
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
-export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+export default function muiThemeable(WrappedComponent, forwardMethods) {
+
+  const methods = {};
+  if (forwardMethods) {
+    forwardMethods.map(name => methods[name] =
+      (function(...args) { return this.refs.WrappedComponent[name](...args); }));
+  }
+
+  return React.createClass(merge({
+    displayName: getDisplayName(WrappedComponent),
+
+    contextTypes: {
+      _muiTheme: React.PropTypes.object,
+    },
 
-  MuiComponent.displayName = getDisplayName(WrappedComponent);
-  MuiComponent.contextTypes = {
-    muiTheme: React.PropTypes.object,
-  };
-  MuiComponent.childContextTypes = {
-    muiTheme: React.PropTypes.object,
-  };
+    childContextTypes: {
+      _muiTheme: React.PropTypes.object,
+    },
```

Comment:
> That's the ThemeProvider's job to provide the context for it's subtree.

I'm not sure about this. @shaurya947 Wanted to be able to use the component without a theme provider.